tests/data/dummy_service_config_files/service_4.yaml

```diff
@@ -0,0 +1,23 @@
+name: dummy_service
+author: valory
+version: 0.1.0
+description: A dummy service config file overrides containing extra parameters.
+aea_version: '>=1.0.0, <2.0.0'
+license: Apache-2.0
+fingerprint:
+  README.md: bafybeiapubcoersqnsnh3acia5hd7otzt7kjxekr6gkbrlumv6tkajl6jm
+fingerprint_ignore_patterns: []
+agent: valory/hello_world:0.1.0:bafybeiaotnukv7oq2sknot73a4zssrrnjezh6nd2fwptrznxtnovy2rusm
+number_of_agents: 4
+---
+benchmark_persistence_params:
+  args: &id001
+    log_dir: /benchmarks
+public_id: valory/hello_world_abci:0.1.0
+type: skill
```

Comment:
This should fail the validation because `benchmark_persistence_params` is extra parameter, but it does not.